tests/autoyast/installation.pm

```diff
@@ -57,7 +57,6 @@ sub save_logs_and_continue {
 
 sub run {
     my $self = shift;
-    $self->result('ok');
```

Comment:
I don't understand. Isn't that implicit? Why would we need to explicitly set this in this test?